cmd/influx_inspect/export/parquet/exporter/meta.go

```diff
@@ -0,0 +1,79 @@
+package exporter
+
+import (
+	"bytes"
+	"encoding/base64"
+	"errors"
+	"io"
+
+	"github.com/influxdata/influxdb/cmd/influx_inspect/export/parquet/models"
+	jsoniter "github.com/json-iterator/go"
+)
+
+// Meta contains information about an exported Parquet file.
+type Meta struct {
+	FirstSeriesKey models.Escaped
+	FirstTimestamp int64
+	LastSeriesKey  models.Escaped
+	LastTimestamp  int64
+}
+
+// meta is an intermediate type to facilitate encoding and decoding
+// series keys in JSON format.
+type meta struct {
+	FirstSeriesKey string `json:"firstSeriesKeyEscaped"`
+	FirstTimestamp int64  `json:"firstTimestamp"`
+	LastSeriesKey  string `json:"lastSeriesKeyEscaped"`
+	LastTimestamp  int64  `json:"lastTimestamp"`
+}
+
+func (m *Meta) UnmarshalJSON(data []byte) error {
+	var tmp meta
+	if err := jsoniter.Unmarshal(data, &tmp); err != nil {
+		return err
+	}
+
+	firstSeriesKey, err := base64.StdEncoding.DecodeString(tmp.FirstSeriesKey)
+	if err != nil {
+		return errors.New("failed to decode firstSeriesKeyEscaped")
```

Comment:
`errors.New` here hides the actual error, which might be helpful in debugging. I think either `errors.Wrap` or `fmt.Errorf` to pass back the root cause error would be more helpful.